### PR TITLE
Add 2.13 release note

### PR DESCRIPTION
**Pull request by zane-neo**
### Description
Add 2.13 release note
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: release-notes/opensearch-skills.release-notes-2.13.0.0.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# 2024-03-20 Version 2.13.0.0
+
+Compatible with OpenSearch 2.13.0
+
```
Comment (owaiskazi19) [inline release-notes/opensearch-skills.release-notes-2.13.0.0.md:4]: We should also mention the [dependencies](https://github.com/opensearch-project/skills/commits/2.13/) bump as under `Upgrades`

Comment (zane-neo) [reply release-notes/opensearch-skills.release-notes-2.13.0.0.md:4]: They're not critical and not features for customers.

Comment (owaiskazi19) [reply release-notes/opensearch-skills.release-notes-2.13.0.0.md:4]: We do that for [OpenSearch](https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch.release-notes-2.12.0.md#dependencies) but upto you.

**Review by owaiskazi19 — APPROVED** [review]

File: release-notes/opensearch-skills.release-notes-2.13.0.0.md (hunk 1 above)
Comment (dhrubo-os) [reply release-notes/opensearch-skills.release-notes-2.13.0.0.md:4]: >We should also mention the [dependencies](https://github.com/opensearch-project/skills/commits/2.13/) bump as under Upgrades

+1 on this. We should add all the PRs in the release notes. May be in a different section like maintenance/upgrades

Comment (zane-neo) [reply release-notes/opensearch-skills.release-notes-2.13.0.0.md:4]: Let's merge this first and I can create another PR to add these.

**Pull request merged**